packages/ui-informed/src/index.ts

```diff
@@ -1,4 +1,31 @@
+/**
+ * FormWithControls
+ */
+export * as FormWithControls from './form';
+
+/**
+ * FormControl
+ */
+export { FormControl } from './controls/formControl/FormControl';
+
 /**
  * Controls
  */
+export { CheckboxControl } from './controls/CheckboxControl';
 export { InputControl } from './controls/InputControl';
+export { SelectControl } from './controls/SelectControl';
+export { TextAreaControl } from './controls/TextAreaControl';
```

Comment:
I would probably export these as names without Suffix. Maybe even drop the `Control` suffix entirely through these codebase (except FormControl). As it is 100% defined by the package itself that these components are controlled by infromed.